### ISSUE TITLE
Final Boss

### ISSUE THREAD

**Issue by Agrair** [Agrair/TheOrderOfSlime#8]
Think of a name. Can resurrect itself multiple times upon death.

**Comment by yepfiery** [comment]
I don’t think it should be a final boss but I’m thinking we should do a deadly 7 sins bosses

**Comment by yepfiery** [comment]
The multiplication boss can be envy

**Comment by Agrair** [comment]
Maybe. Like they’re all gods or something? Or an event? Or maybe we could be the final bosses with the dev armour we still have to make.

**Comment by Agrair** [comment]
no I meant they are wearing the dev things
